modules/dqt/jsx/react.app.js

```diff
@@ -16,6 +16,7 @@ import {StepperPanel, ProgressBar} from './components/stepper';
 import SavedQueriesList from './react.savedqueries';
 import ExpansionPanels from './components/expansionpanels';
 import NoticeMessage from './react.notice';
+import {getSessions} from '../js/arrayintersect';
```

Comment:
@ridz1208 this import fixes the "Fixes the unable to load a saved query bug. One of the functions needed wasn't being imported in the react.app.js" bug.